sdk/servicebus/service-bus/package.json

```diff
@@ -44,21 +44,29 @@
     "@types/debug": "^0.0.31",
     "@types/dotenv": "^6.1.0",
     "@types/mocha": "^5.2.5",
-    "@types/node": "^8.0.0",
+    "@types/node": "^8.0.37",
```

Comment:
Please revert this change, this should be pinned at ^8.0.0

Reply:
done